index.html

```diff
@@ -206,27 +206,41 @@
 the more accurate "an object implementing the interface <code
 title="">Foo</code>". </p>
 
-<p>The term DOM is used to refer to the API set made available to scripts in
+<p>The term <dfn>DOM</dfn> is used to refer to the API set made available to scripts in
 Web applications, and does not necessarily imply the existence of an actual
 <code>Document</code> object or of any other <code>Node</code> objects as
 defined in the DOM specification. [[DOM]]
 </p>
 
-<p>A DOM attribute is said to be <em>getting</em> when its value is being
-retrieved (such as by author script), and is said to be <em>setting</em> when
+<p>A DOM attribute is said to be <dfn>getting</dfn> when its value is being
+retrieved (such as by author script), and is said to be <dfn>setting</dfn> when
 a new value is assigned to it.</p>
 
-<p>The term "JavaScript" is used to refer to ECMA262, rather than the
+<p>The term <dfn>JavaScript</dfn> is used to refer to ECMA262, rather than the
 official term ECMAScript, since the term JavaScript is more widely known. [[ECMA-262]]
 </p>
 
+<p>The term <dfn>resource</dfn> is used to refer to these elements and any other user-initiated fetches throughout this specification.</p>

```

Comment:
My goal here was only to make sure all term definitions were in the terminology sections. But it does surface the fact that our definitions aren't perfect indeed.